pydsa/tests/test_binary_search.py

```diff
@@ -1,12 +1,19 @@
 from pydsa.binary_search import binary_search
 from random import randint
 
+
```

Comment:
According to pep8 conventions there should be two blank lines between import statements and code.